### PR TITLE
Change icons to use drawable reference

### PR DESCRIPTION
**Pull request by wmontwe**
This removes the attr definitions for icons as we're using the same ones for dark and light mode and there is no reason to keep this setup in place. The icons are now referenced directly, which will allow easier transition to the new Material 3 icons and consolidating them into it's own icon module.

`FoldableLinearLayout` was removed as it was only referenced by the attr definition.

 This change allowed to remove some unused icons.

### CODE REVIEW

**Review by cketti — APPROVED** [review]
I liked having a layer to separate between icon (`ic_cog`) and function (`iconSettings`) and hope we'll bring it back in some form at some point.

**Comment by wmontwe** [comment]
> I liked having a layer to separate between icon (`ic_cog`) and function (`iconSettings`) and hope we'll bring it back in some form at some point.

We discussed this already with design and we are going to use the icon names from the Material icon library.

File: app/ui/legacy/sampledata/folders.json
[the remark this answers is not in the document]
Comment (wmontwe) [reply app/ui/legacy/sampledata/folders.json:1]: That was throwing some exceptions.

File: app/ui/legacy/src/main/java/com/fsck/k9/ui/messagelist/MessageListAdapter.kt [hunk 1]
```diff
@@ -51,9 +50,10 @@ class MessageListAdapter internal constructor(
     private val relativeDateTimeFormatter: RelativeDateTimeFormatter,
 ) : RecyclerView.Adapter<MessageListViewHolder>() {
 
-    private val forwardedIcon: Drawable = theme.resolveDrawableAttribute(R.attr.messageListForwarded)
-    private val answeredIcon: Drawable = theme.resolveDrawableAttribute(R.attr.messageListAnswered)
-    private val forwardedAnsweredIcon: Drawable = theme.resolveDrawableAttribute(R.attr.messageListAnsweredForwarded)
+    private val forwardedIcon: Drawable = ResourcesCompat.getDrawable(res, R.drawable.ic_messagelist_forwarded, theme)!!
+    private val answeredIcon: Drawable = ResourcesCompat.getDrawable(res, R.drawable.ic_messagelist_answered, theme)!!
+    private val forwardedAnsweredIcon: Drawable = ResourcesCompat
+        .getDrawable(res, R.drawable.ic_messagelist_answered_forwarded, theme)!!
```
[the remark this answers is not in the document]
Comment (wmontwe) [reply app/ui/legacy/src/main/java/com/fsck/k9/ui/messagelist/MessageListAdapter.kt:56]: That could be a separate task as the `MessageListAdapter` is quite complex.

**Pull request merged**